### DISCUSSION TITLE
can we make run it on virtualbox?

### DISCUSSION THREAD

**Discussion by omerstudio** [CosmosOS/Cosmos#2595] [Q&A]
i just dont wanna run on vmware, i wanna run it on virtualbox too. could please someone help?

**Comment by Arawn-Davies** [comment]
I think one of the only blockers on this would be the debugger, but that should be fairly extendable to include VirtualBox as well.
Saying that, I think sometimes VirtualBox is buggy with Cosmos (mainly graphics and filesystems) but these issues might have been resolved since I last checked

**Reply by omerstudio** [reply]
Thanks for the information.

**Reply by quajak** [reply]
I don't think there has been any progress on the stability but if we would have better debugging setup on virtual box, that would be easier to fix,
